dev-python/sentry-sdk/sentry-sdk-0.14.4-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{6..8})
+
+inherit distutils-r1
+
+DESCRIPTION="Python client for Sentry"
+HOMEPAGE="https://getsentry.com https://pypi.org/project/sentry-sdk/ https://github.com/getsentry/sentry-python"
+SRC_URI="https://github.com/getsentry/sentry-python/archive/${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
+
+S="${WORKDIR}/sentry-python-${PV}"
+
+RDEPEND="
+	dev-python/urllib3[${PYTHON_USEDEP}]
+	dev-python/certifi[${PYTHON_USEDEP}]
+"
+
+BDEPEND="${RDEPEND}
+	test? (
+		dev-python/pytest-asyncio[${PYTHON_USEDEP}]
+		dev-python/pytest-forked[${PYTHON_USEDEP}]
+		dev-python/pytest-localserver[${PYTHON_USEDEP}]
+		dev-python/eventlet[${PYTHON_USEDEP}]
+		dev-python/gevent[${PYTHON_USEDEP}]
+		dev-python/werkzeug[${PYTHON_USEDEP}]
+	)
+"
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	# reason for removal:
+	# test_requests.py - requires internet access
+	# test_httplib.py - requires internet access
+	# test_transport.py - failing with ebuild somehow, running tests directly via pytest works
+	# django integration tests - failing, also upstream failing when called directly - via tox these are working
+	if use test; then
+		rm --recursive \
+			"${S}"/tests/integrations/requests/test_requests.py \
+			"${S}"/tests/integrations/stdlib/test_httplib.py \
+			"${S}"/tests/test_transport.py \
+			"${S}"/tests/integrations/django
+
+		# disable-auto-enabling-intregrations-tests
+		sed --in-place --expression='s:test_auto_enabling_integrations_catches_import_error:_&:' \
+			tests/test_basics.py
```

Comment:
|| die

Also please use short sed args, and try to keep lines under 72/80 chars.

Reply:
Changed